### PR TITLE
Add module for import BRD MT940bank statements.

### PR DESCRIPTION
**Pull request by feketemihai**

### CODE REVIEW

File: account_bank_statement_import_mt940_ro_brd/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Import MT940 ROMANIAN BRD Bank Statements
```
Comment (pedrobaeza) [inline account_bank_statement_import_mt940_ro_brd/README.rst]: Put a line above

File: account_bank_statement_import_mt940_ro_brd/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (pedrobaeza) [inline account_bank_statement_import_mt940_ro_brd/README.rst]: Use latest template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: account_bank_statement_import_mt940_ro_brd/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# ©  2016 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+{
+    'name': 'MT940 BRD Format Bank Statements Import',
+    'version': '8.0.0.1.0',
```
Comment (pedrobaeza) [inline account_bank_statement_import_mt940_ro_brd/__openerp__.py]: 8.0.1.0.0

File: account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940 [hunk 4]
```diff
@@ -0,0 +1,17 @@
+:20:6450374100
```
Comment (pedrobaeza) [inline account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940]: Why not put this in the same tests folder?

File: oca_dependencies.txt [hunk 5]
```diff
@@ -1,8 +1,10 @@
 # Add account-financial-tools repo dependency to have access to currency_rate_update module.
 # Add account-payment repo dependency to have access to account_vat_on_payment and account_voucher_cash_basis modules.
+# Add bank-statement-import repo dependency to have access to MT940 import statements modules.
```
Comment (pedrobaeza) [inline oca_dependencies.txt]: You don't need to put these comments in the file.

File: oca_dependencies.txt [hunk 6]
```diff
@@ -1,8 +1,10 @@
...
 # Add hr repo dependency to have access to some hr modules.
 # Add partner-contact repo dependency to have access to partner_create_by_vat module.
 account-financial-tools https://github.com/OCA/account-financial-tools 8.0
 account-payment https://github.com/OCA/account-payment 8.0
+bank-statement-import https://github.com/OCA/bank-statement-import 8.0
```
Comment (pedrobaeza) [inline oca_dependencies.txt]: The file can be extremely simplified in this case, as the organization and branch is the same:

```
bank-statement-import
hr
partner-contact
...
```

File: account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940 (hunk 4 above)
Comment (feketemihai) [inline account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940]: In the https://github.com/OCA/bank-statement-import/blob/8.0/account_bank_statement_import/tests/test_import_file.py#L90 there is the path predefined to the folder "test_files" from the module path...

File: oca_dependencies.txt (hunk 6 above)
Comment (feketemihai) [inline oca_dependencies.txt]: @pedrobaeza  And is taken directly the 8.0 branch, even if the 9.0 is the default one?

File: account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940 (hunk 4 above)
Comment (pedrobaeza) [inline account_bank_statement_import_mt940_ro_brd/test_files/test-brd.940]: Ah, OK, I made the tests independently from this: https://github.com/OCA/l10n-spain/blob/8.0/l10n_es_account_bank_statement_import_n43/tests/test_l10n_es_account_bank_statement_import_n43.py

File: oca_dependencies.txt (hunk 6 above)
Comment (pedrobaeza) [inline oca_dependencies.txt]: Yes, the taken branch is the same as the one you are making the changes.

**Comment by pedrobaeza** [comment]
You should called the module _l10n_ro_account_bank_statement_import_mt940_brd_

**Comment by feketemihai** [comment]
@pedrobaeza Can i have your last review here, please...

File: l10n_ro_account_bank_statement_import_mt940_brd/tests/test_import_bank_statement.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+class TestImport(TestStatementFile):
+    """Run test to import MT940 ING import."""
+
+    def test_statement_import(self):
+        """Test correct creation of single statement."""
+        self._test_statement_import(
+            'l10n_ro_account_bank_statement_import_mt940_brd', 'test-brd.940',
+            '00138/1',
+            start_balance=3885.24, end_balance=3671.88
+        )
```
Comment (pedrobaeza) [inline l10n_ro_account_bank_statement_import_mt940_brd/tests/test_import_bank_statement.py]: You should make at least basic assert like the number of lines read, and some comparison between hardcoded amounts versus read amounts.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
You should make at least basic assert like the number of lines read, and some comparison between hardcoded amounts versus read amounts.

**Comment by feketemihai** [comment]
@pedrobaeza Updated test, thanks a lot for reviewing in short time. Thanks again.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**